### PR TITLE
Hook it up the new datatransfer server.

### PR DESCRIPTION
**Pull request by Gerbenvandervries**
Hook it up the new datatransfer server.

### CODE REVIEW

File: shell/synchronize/etc/ditf.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
-source_dir='/groups/solve-rd/prm10/sftp/ditf'
-destination_dir='/groups/umcg-solve-rd/scr01/'
-destination_machine='umcg-${ATEAMBOTUSER}@cher-ami.hpc.rug.nl'
+ksource_dir='/groups/solve-rd/prm10/sftp/ditf'
```
Comment (scimerman) [inline shell/synchronize/etc/ditf.cfg:1]: Is this correct, or should it be
```suggestion
source_dir='/groups/solve-rd/prm10/sftp/ditf'
```
?

File: shell/synchronize/etc/datf-share.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
-source_dir="/groups/umcg-solve-rd/scr01/datf/${data_group}/share"
+source_dir="groups/umcg-solve-rd/datf/${data_group}/share"
```
Comment (pneerincx) [inline shell/synchronize/etc/datf-share.cfg:1]: Are you sure the / at the front and changing the absolute into a relative path needs to be removed?

File: shell/synchronize/etc/datf-share.cfg [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 destination_dir="/groups/solve-rd/prm10/sftp/datf/${data_group}/"
-destination_machine='umcg-${ATEAMBOTUSER}@cher-ami.hpc.rug.nl'
+destination_machine='nb-dt'
```
Comment (pneerincx) [inline shell/synchronize/etc/datf-share.cfg:3]: The machine was also renamed in the inventory nb-dt -> nb-transfer to match the DNS entry.

File: shell/synchronize/etc/datf-share.cfg (hunk 2 above)
Comment (pneerincx) [reply shell/synchronize/etc/datf-share.cfg:1]: Never mind, you are now using rsync module syntax, which explains why the leading / had to go.

File: shell/synchronize/etc/datf.cfg [hunk 4]
```diff
@@ -1,6 +1,6 @@
 source_dir='/groups/solve-rd/prm10/sftp/datf'
-destination_dir='/groups/umcg-solve-rd/scr01/'
-destination_machine='umcg-${ATEAMBOTUSER}@cher-ami.hpc.rug.nl'
+destination_dir='groups/umcg-solve-rd/'
```
Comment (scimerman) [inline shell/synchronize/etc/datf.cfg:2]: Like Pieter mentioned on the other file. Is this line with missing starting '/' correct?

**Review by pneerincx — CHANGES_REQUESTED** [review]
nb-dt -> nb-transfer. See inline comment.

File: shell/synchronize/etc/datf.cfg (hunk 4 above)
Comment (pneerincx) [reply shell/synchronize/etc/datf.cfg:2]: Yes, is correct (rsync "module" syntax).

**Review by pneerincx — APPROVED** [review]

**Pull request merged**